web/filipino_cuisine/web/data.json

```diff
@@ -0,0 +1 @@
 cook for 3 minutes more","Transfer to a serving plate. Serve"],"pf":"beef_caldereta.jpg","pt":"10 minutes","sv":"3","tt":"13 MIN"},"4":{"cn":"90 Gr     130 Kcal     1 Hour 15 Min","ct":"3 minutes","dc":"Pork Embutido is a Filipino-style meatloaf made with a festive mixture of ground pork, carrots, and raisins wrapped around slices of eggs and sausage.","fn":"Embutido","fv":false,"ig":[{"c":"2 lbs","n":"Ground pork","p":"ground_pork.png"},{"c":"12 pcs","n":"Sausage","p":"sausage.png"},{"c":"3 pcs","n":"Egg","p":"egg.png"},{"c":"2 cups","n":"Cheese","p":"cheese.png"},{"c":"1 cup","n":"Red Bell Pepper","p":"red_bell.png"},{"c":"1 cup","n":"Green Bell Pepper","p":"green_bell.png"},{"c":"1/2 cup","n":"Raisins","p":"raisins.png"},{"c":"1/2 cup","n":"Carrot","p":"carrot.png"},{"c":"1/2 cup","n":"Onion","p":"onion.png"},{"c":"2 tbsp","n":"Salt","p":"salt.png"},{"c":"1 tbsp","n":"Peppercorn","p":"black_pepper.png"}],"in":["Place the ground pork in a large container","Add the bread crumbs then break the raw eggs and add it in. Mix well","Put-in the carrots, bell pepper (red and green), onion, pickle relish, and cheddar cheese. Mix thoroughly","Add the raisins, tomato sauce, salt, and pepper then mix well.","Put in the sliced vienna sausage and sliced boiled eggs alternately on the middle of the flat meat mixture.","Roll the foil to form a cylinder — locking the sausage and egg in the middle if the meat mixture. Once done, lock the edges of the foil.","Place in a steamer and let cook for 1 hour.","Place inside the refrigerator until temperature turns cold","Slice and serve"],"pf":"embutido.jpg","pt":"10 minutes","sv":"3","tt":"13 MIN"},"-M6-NHS2k032b2T5Dx-u":{"amount":"150.00","cardPan":"4111XXXXXXXX1111","currency":"BGN","date":"2019-09-27 13:00:00","operation":"payment","payneticsToken":"gCZ7v3HnoeX5LLlQuASETRb0gDKkANbAF21Xc5__3XM","reference":"65165","status":1,"successfulPayment":true,"transaction":"00000000007"},"-M6-PG01yTAMSepR-GDE":{"amount":"150.00","cardPan":"4111XXXXXXXX1111","currency":"BGN","date":"2019-09-27 13:00:00","operation":"payment","payneticsToken":"gCZ7v3HnoeX5LLlQuASETRb0gDKkANbAF21Xc5__3XM","reference":"65165","status":1,"successfulPayment":true,"transaction":"00000000007"},"-M6-Pf-WdBL9fzBlCmLn":{"amount":"150.00","cardPan":"4111XXXXXXXX1111","currency":"BGN","date":"2019-09-27 13:00:00","operation":"payment","payneticsToken":"gCZ7v3HnoeX5LLlQuASETRb0gDKkANbAF21Xc5__3XM","reference":"65165","status":1,"successfulPayment":true,"transaction":"00000000007"},"-M6-UCy5f43Nc8JUwDWl":{"amount":"150.00","cardPan":"4111XXXXXXXX1111","currency":"BGN","date":"2019-09-27 13:00:00","operation":"payment","payneticsToken":"gCZ7v3HnoeX5LLlQuASETRb0gDKkANbAF21Xc5__3XM","reference":"65165","status":1,"successfulPayment":true,"transaction":"00000000007"},"-M64U95I5C3vXg_kR0ID":{"login":{"auth":{"systemCode":"ZLH4WEtjbwA8vPOAJ5XTwdXcGpr7rxBC00/Q2JX7Dh4USg0mYApAQFd+a6CLcf3C9oStO/gUxlM=","systemId":1050},"password":"Tbi*10011","username":"8033 Таня"}}}
```

Comment:
What's the data at the end of this JSON used for (the part beginning "systemCode")?

Reply:
not sure, I'll take that out.